src/modules/recorder.module.ts

```diff
@@ -308,6 +310,48 @@ async function handleDeleteClip(clipName: string, requester: string, textChannel
     }
 }
 
+async function handleTrimClip(clipName: string, start: number, end: number, textChannel: TextChannel) {
+    let clip
+    try {
+        clip = await getClipByName(clipName)
+    } catch (err) {
+        return await textChannel.send("Could not find a clip with that name.")
+    }
+
+    const duration = offsetToSeconds(clip.duration)
+
+    // if end was not passed in (NaN), assume end === duration
+    if (!end) {
+        end = duration
+    }
+
+    // convert negative numbers to (duration - n)
+    if (start < 0) {
+        start = duration + start
+    }
+
+    if (end < 0) {
+        end = duration + end
+    }
+
+    // reject if start > duration
+    if (start > duration) {
+        return await textChannel.send(`Start time is after the end of the clip (${duration.toFixed(1)})`)
+    }
+
+
+    if (start >= end) {
+        return await textChannel.send("Start time must be before end time.")
+    }
+

```

Comment:
What happens when `start < -duration` and `end < -duration`? How does this treat still negative values?